rbd.c

```diff
@@ -254,7 +262,8 @@ static int tcmu_rbd_lock_break(struct tcmu_device *dev, char **orig_owner)
 
 	if (ret < 0) {
 		tcmu_dev_err(dev, "Could not get lock owners %d\n", ret);
-		return -EAGAIN;
+		if (ret != -ETIMEDOUT)
```

Comment:
If you time-out attempting to get the lock owners, I'd imagine you'd either need to retry or fail out of this function instead of continuing like out-params are valid.